cpp/simulations/abm.cpp

```diff
@@ -401,27 +418,25 @@ void create_assign_locations(mio::abm::World& world)
         std::vector<mio::abm::TestingCriteria>{mio::abm::TestingCriteria({}, test_at_school, {})};
 
     testing_min_time           = mio::abm::days(7);
-    probability                = 1.0;
     auto testing_scheme_school = mio::abm::TestingScheme(testing_criteria_school, testing_min_time, start_date,
-                                                         end_date, test_type, probability);
+                                                         end_date, test_type, mio::UncertainValue(1.0));
     world.get_testing_strategy().add_testing_scheme(testing_scheme_school);
 
     auto test_at_work = std::vector<mio::abm::LocationType>{mio::abm::LocationType::Work};
     auto testing_criteria_work =
         std::vector<mio::abm::TestingCriteria>{mio::abm::TestingCriteria({}, test_at_work, {})};
 
-    testing_min_time = mio::abm::days(1);
-    probability      = 0.5;
-    auto testing_scheme_work =
-        mio::abm::TestingScheme(testing_criteria_work, testing_min_time, start_date, end_date, test_type, probability);
+    testing_min_time         = mio::abm::days(1);
+    auto testing_scheme_work = mio::abm::TestingScheme(testing_criteria_work, testing_min_time, start_date, end_date,
+                                                       test_type, mio::UncertainValue(0.5));
```

Comment:
See above.

Reply:
I change it to have a probability between 0.1-0.5 and can be drawn at the beginning of each simulation.